node/rpc-api/src/data_types.rs

```diff
@@ -186,9 +188,9 @@ pub struct BlockTemplate {
 #[derive(Serialize)]
 #[serde(rename_all = "PascalCase")]
 pub struct MiningBaseInfoJson {
-    #[serde(with = "bigint_ser::json::opt")]
+    #[serde(with = "opt")]
```

Comment:
nit: Maybe rename `opt` to `option` and keep the `json` prefix. Like this: `#[serde(with = "json::option")]`

Reply:
Fix